GameServerLib/API/ApiFunctionManager.cs

```diff
@@ -416,14 +416,7 @@ public static IAttackableUnit GetClosestUnitInRange(Vector2 targetPos, float ran
         /// <returns>Closest AttackableUnit.</returns>
         public static IAttackableUnit GetClosestUnitInRange(IAttackableUnit target, float range, bool isAlive)
         {
-            var units = _game.ObjectManager.GetUnitsInRange(target.Position, range, isAlive);
-            var orderedUnits = units.OrderBy(unit => Vector2.DistanceSquared(target.Position, unit.Position));
-            if (orderedUnits.First() == target)
-            {
-                return orderedUnits.ElementAt(1);
-            }
-
-            return orderedUnits.First();
+            return GetClosestUnitInRange(target.Position, range, isAlive);
```

Comment:
Sorry if I wasn't clear. Just the 2 first lines can be replaced. You still have to check that you don't retrieve target. (at least this is the `if` that happens here)